### PR TITLE
Fixes IntelliSense crash.

### PR DESCRIPTION
**Pull request by jas-valgotar**

### CODE REVIEW

**Review by anderson-joyle — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionHandlers/FunctionRecordNameSuggestionHandler.cs [hunk 1]
```diff
@@ -197,7 +197,7 @@ internal static DType GetAggregateType(TexlFunction func, CallNode callNode, Int
                 }
                 else
                 {
-                    type = intellisenseData.Binding.GetType(callNode.Args.Children[0]);
+                    type = callNode.Args?.Count > 0 ? intellisenseData.Binding.GetType(callNode.Args.Children[0]) : DType.Unknown;
```
[the remark this answers is not in the document]
Comment (jas-valgotar) [reply src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionHandlers/FunctionRecordNameSuggestionHandler.cs]: sure we could do that, updated! :)

**Review by anderson-joyle — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**